core/config/initializers/27_sitemaps.rb

```diff
@@ -0,0 +1,3 @@
+# Quiet the output of the generator in tests to eliminate
+# the noise when running the test suite.
+SitemapGenerator.verbose = !Rails.env.test?
```

Comment:
YAS

Reply:
not sure why we didn't do this before

Reply:
i literally had no idea this existed otherwise i definitely would have!